packages/oasis-actions/src/operations/aave/close.ts

```diff
@@ -93,9 +100,12 @@ export async function close(
       paybackInAAVE,
       withdrawDAIFromAAVE,
       unwrapEth,
-      returnFunds,
+      returnDebtFunds,
+      returnCollateralFunds,
     ],
   })
 
-  return { calls: [takeAFlashLoan], operationName: 'CUSTOM_OPERATION' }
+  const operationName = OPERATION_NAMES.aave.CLOSE_POSITION
```

Comment:
Is assigning to a new variable needed here? If so, then you can do `return { operationName, calls: [takeAFlashLoan] }`